docs/Tutorials/Private-Network/Adding-removing-IBFT-validators.md

```diff
@@ -0,0 +1,118 @@
+---
+description: Adding and removing IBFT 2.0 validators
+---
+
+# Adding and removing IBFT 2.0 validators
+
+This example walks through
+[adding and removing an IBFT 2.0 validator by voting](../../HowTo/Configure/Add-Validators.md#adding-and-removing-validators-by-voting).
+
+## Prerequisites
+
+* [IBFT 2.0 network as configured in the IBFT 2.0 tutorial](Create-IBFT-Network.md)
+
+## Adding a validator
+
+### 1. Create directories and start the validator
+
+Create a working directory and a data directory for the new node that needs to be added:
+
+```bash
+mkdir -p Node-5/data
+```
+
+Change into the working directory for the new Node-5 and start the node, specifying the
+[Node-1 enode URL](Create-IBFT-Network.md#6-start-the-first-node-as-the-bootnode) as the bootnode:
+
+```bash
+besu --data-path=data --genesis-file=../genesis.json --bootnodes=<Node-1 Enode URL> --p2p-port=30304 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-allowlist="*" --rpc-http-cors-origins="all" --rpc-http-port=8546
```

Comment:
You may want to test this, but I think you may end up with some clashing ports here. Will probably clash with Node 2.